src/modules/common/components/FormattedNumber/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './index';
```

Comment:
@soroushm The index file exports itself rather than the FormattedNumber component